optuna/structs.py

```diff
@@ -275,13 +263,18 @@ def duration(self):
 class StudySummary(object):
```

Comment:
We should probably either rely on `StudyDirection = _study_direction.StudyDirection` or define this class, but not both.